src/InteractiveWindow/Version.props

```diff
@@ -1,5 +1,5 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <RoslynSemanticVersion>1.0.0</RoslynSemanticVersion>
+    <RoslynSemanticVersion>1.1.0</RoslynSemanticVersion>
```

Comment:
More importantly, since we down depend on this version in order for our Repl commands to work, rev'ing the version allows us to express the dependency (hopefully making it easier to avoid or subsequently diagnose horked machine state).